src/libs/PolicyUtils.ts

```diff
@@ -0,0 +1,217 @@
+import Str from 'expensify-common/lib/str';
+import CONST from '../CONST';
+import ONYXKEYS from '../ONYXKEYS';
+import * as OnyxTypes from '../types/onyx';
+
+type PolicyMemberList = Record<string, OnyxTypes.PolicyMember>;
+type PolicyMembersCollection = Record<string, PolicyMemberList>;
+type MemberEmailsToAccountIDs = Record<string, string>;
+type PersonalDetailsList = Record<string, OnyxTypes.PersonalDetails>;
+type UnitRate = {rate: number};
+
+/**
+ * Filter out the active policies, which will exclude policies with pending deletion
+ * These are policies that we can use to create reports with in NewDot.
+ */
+function getActivePolicies(policies: OnyxTypes.Policy[]): OnyxTypes.Policy[] {
```

Comment:
Double check that it is indeed an array, from the usage I think it actually might be an object @BartoszGrajdek

Reply:
Same for other functions